src/app/profile/settings/feature-opt-in/feature-opt-in.component.less

```diff
@@ -1,58 +1,51 @@
 @import (reference) '../../../../assets/stylesheets/shared/osio.less';
 
-.feature-title {
-  font-weight: bold;
-  font-size: @font-size-base;
-  margin-bottom: 0;
-}
-
-.feature-description {
-  margin-bottom: 0;
-  font-size: .85em;
-}
-
-.feature-details-title {
-  margin-bottom: 7px;
-  font-size: @font-size-base;
-  font-weight: 600;
-}
-
-.feature-details-description {
-  margin-bottom: 1em;
-  font-size: @font-size-small;
-  ul {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-    line-height: 2em;
+.feature-opt-in {
+  .checkbox {
+    border-right: 1px solid @color-pf-black-300;
+    float: left;
+    margin-top: 20px;
+    margin-right: 15px;
+    margin-bottom: 20px;
+    padding: 3px 10px 3px 0;
+  }
```

Comment:
The checkbox selector raised a red flag for me. The list-pf-content selector of pfng-list already provides a border when useExpandRows is enabled.

It appears that you have implemented your own checkbox and expansion toggle. Why use the pfng-list at all if just bypassing all of its features?